python/lang/security/deserialization/avoid-pyyaml-load.yaml

```diff
@@ -36,5 +36,10 @@ rules:
       - pattern-not: yaml.load_all(..., Loader=yaml.SafeLoader, ...)
       - pattern-not: yaml.load_all(..., Loader=yaml.BaseLoader, ...)
       - pattern-either:
-          - pattern: yaml.load(...)
-          - pattern: yaml.load_all(...)
+          - pattern: yaml.unsafe_load(...)
```

Comment:
These patterns look good! Thanks :D

The `pattern-not`s above are technically not necessary anymore. They won't hurt anything, but it might be good to remove them for readability.

Reply:
I elected to remove them and include the safe patterns in the rule message instead. Thanks! 🙏

Reply:
ok looks good